rllib/examples/ray_serve/classes/cartpole_deployment.py

```diff
@@ -21,22 +21,30 @@ class ServeRLlibRLModule:
       (with a current observation).
     """
 
-    def __init__(self, checkpoint):
-        self.algo = Algorithm.from_checkpoint(checkpoint)
+    def __init__(self, rl_module_checkpoint):
+        self.rl_module = RLModule.from_checkpoint(rl_module_checkpoint)
 
     async def __call__(self, starlette_request: Request) -> Dict:
         request = await starlette_request.body()
         request = request.decode("utf-8")
         request = json.loads(request)
         obs = request["observation"]
 
-        # Compute and return the action for the given observation.
-        action = self.algo.compute_single_action(obs)
+        # Compute and return the action for the given observation (create a batch
+        # with B=1 and convert to torch).
+        output = self.rl_module.forward_inference(
```

Comment:
For a better practical usability we might need to enlarge this example by a connector run. This might be the default we will see in practice.

Reply:
Yeah, makes sense!
Maybe add another example where we restore the EnvToModule pipeline + the RLModule.
Like we do in the `ray.rllib.examples.inference.policy_inference_after_training_w_connector.py` script.